internal/bootstrap/bootstrap_config.go

```diff
@@ -846,7 +846,8 @@ func appendTelemetryCollectorConfig(args *BootstrapTplArgs, telemetryCollectorBi
 			"envoy_grpc": {
 			  "cluster_name": "consul_telemetry_collector_loopback"
 			}
-		  }
+		  },
+		  "emit_tags_as_labels": true
```

Comment:
There's another PR up which adds these: https://github.com/hashicorp/consul-dataplane/pull/184. I'd recommend we merge that one first so that it has the right changelog entry (I approved) and then rebase this PR

Reply:
sure. will wait for it be merged.

Reply:
I just merged https://github.com/hashicorp/consul-dataplane/pull/184, thanks for waiting ya'll